### PR TITLE
XSUP-37130/AbnormalEventCollector

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: https://jira-dc.paloaltonetworks.com/browse/XSUP-37130

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by eyalpalo — APPROVED** [review]

**Comment by sapirshuker** [comment]
Validate and pre-commit failed on:
`[ERROR] Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector.yml: [IN124] - Parameter: "after" can't be hidden in all marketplaces. Please either remove the `hidden` attribute, or replace its value with a list of marketplace names, where you wish it to be hidden.`

File: Packs/AbnormalSecurity/ReleaseNotes/2_2_9.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Abnormal Security Event Collector
+
+- Changed the default value for *First fetch time interval* parameter to 1 minute and hide it.
```
Comment (ShirleyDenkberg) [inline Packs/AbnormalSecurity/ReleaseNotes/2_2_9.md:6]:
```suggestion
- Changed the default value for *First fetch time interval* parameter to 1 minute and hid it.
```

**Comment by ShirleyDenkberg** [comment]
@eyalpalo Doc review completed.

**Comment by sapirshuker** [comment]
pre-commit and validate error: 
`[ERROR] Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector.yml: [IN124] - The following fields are hidden and cannot be hidden, please unhide them: after.`

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def main():
     verify = params['verify']
     proxy = params['proxy']
     after = arg_to_datetime(
-        arg=params.get('after'), arg_name='after', required=True).strftime("%Y-%m-%dT%H:%M:%SZ")  # type: ignore
+        arg='1 minute', arg_name='after', required=True).strftime("%Y-%m-%dT%H:%M:%SZ")  # type: ignore
```
Comment (eyalpalo) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector.py:114]: If we have 1 minute hard coded we do not need the arg_name nor the required

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**